index.html

```diff
@@ -6581,6 +6568,12 @@ <h3>Executing Script</h3>
 
    <li><p>Otherwise, return <var>cloned property result</var>.
   </ol>
+
+  <li>
+    <p>Remove the last element of <var>seen</var>.
```

Comment:
Generally we keep the `<p>` on the same line as the `<li>`

Reply:
I'll fix that.